python/paddle/v2/fluid/tests/book_distribute/notest_dist_image_classification.py

```diff
@@ -1,16 +1,16 @@
-#Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserved
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
It should be `Reserved` instead of `Reserve`. Check -> https://en.wikipedia.org/wiki/All_rights_reserved

Reply:
Thanks so much, I will update it for all the files then.